project.clj

```diff
@@ -4,5 +4,5 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :repositories [["choco.repos" "http://www.emn.fr/z-info/choco-repo/mvn/repository/"]]
-  :dependencies [[org.clojure/clojure "1.5.1"]
+  :dependencies [[org.clojure/clojure "1.7.0-beta1"]
```

Comment:
If you want to upgrade to 1.6.0 that's fine, but for now I'd like to stay on stable versions of Clojure until the official 1.7.0 is released. :)